### DISCUSSION TITLE
Update release date

### DISCUSSION THREAD

**Discussion by Fernyam** [ldtteam/minecolonies#10506] [Q&A] [answered]
Hello, I'd like to ask. When will the update with the expedition system be released and will it be on 1.20.1?

**Comment by Thodor12** [comment]
We do not commit to specific release dates for anything. It's currently in Patreon testing, no clue when it's going out.

**Comment by Fernyam** [comment]
Thanks, but will this feature be available on 1.20.1?

**Comment by Thodor12** [comment, marked as answer]
It will be made for 1.20.1 first, then ported over to 1.21 as this requires some significant changes

**Comment by Fernyam** [comment]
Well
